lib/MutationOperators/AddMutationOperator.cpp

```diff
@@ -202,7 +202,7 @@ llvm::Value *AddMutationOperator::applyMutation(Module *M,
   Instruction *replacement = BinaryOperator::Create(Instruction::Sub,
                                                     binaryOperator->getOperand(0),
                                                     binaryOperator->getOperand(1),
-                                                    binaryOperator->getName());
+                                                    "sub");
```

Comment:
Point was to make mutations explicit. I also changed it to make my unit test more explicit but no specific point I'm making. Reverted.